templates/partial/contact_info.html

```diff
@@ -3,9 +3,12 @@
 <div class="govuk-body">
   <h2 class="govuk-heading-s govuk-!-margin-bottom-1">{% translate "Access requirements" %}</h2>
   <p id="request_access" class="govuk-body">
+
     {% if access_requirements %}
       {% if is_access_url %}
         <a onclick="gtag('event', 'click_access_link', {'entity_name': '{{ entity_name }}'})" href="{{ access_requirements }}" class="govuk-link" rel="noreferrer noopener" target="_blank">{% translate "Click link for access information (opens in new tab)" %}</a>
+      {% elif entity.external_url %}
```

Comment:
As discussed on slack, this needs to be reconciled with the chart template. This is the ticket in the backlog I mentioned: https://github.com/ministryofjustice/find-moj-data/issues/951